doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -259,13 +259,19 @@ Bluetooth Mesh samples
 Cellular samples
 ----------------
 
+* :ref:`location_sample` sample:
+
+  * Removed nRF7002 EK DTC overlay file ``nrf91xxdk_with_nrf7002ek.overlay`` because disabling UART1 is done through the shield configuration.
+
 * :ref:`modem_shell_application` sample:
 
   * Added:
 
     * Printing of the last reset reason when the sample starts.
     * Support for printing the sample version information using the ``version`` command.
 
+  * Removed nRF7002 EK DTC overlay file ``nrf91xxdk_with_nrf7002ek.overlay`` because disabling UART1 is done through the shield configuration.
```

Comment:
```suggestion
  * Removed nRF7002 EK DTC overlay file ``nrf91xxdk_with_nrf7002ek.overlay`` because UART1 is disabled through the shield configuration.
```